setup.py

```diff
@@ -129,41 +128,36 @@ def py2exe_options():
     author_email=author_email,
     classifiers=filter(None, classifiers.split('\n')),
     license=license,
-    scripts=['openerp-server', 'odoo.py'],
+    entry_points={
+        'console_scripts': [
+            'odoo=odoo.cli:main',
+        ]
+    },
     packages=find_packages(),
-    package_dir={'%s' % lib_name: 'openerp'},
+    package_dir={'%s' % lib_name: 'odoo'},
     include_package_data=True,
     install_requires=[
         'babel >= 1.0',
         'decorator',
         'docutils',
         'feedparser',
-        'gevent',
```

Comment:
Nope, it's required if you run in workers mode. Psycogreen is not packaged in Debian Jessie so we can not put it in the install_requires (as the console_script will check that).